app/apollo/resolvers/cluster.js

```diff
@@ -386,12 +427,38 @@ const clusterResolvers = {
       try {
         logger.debug({req_id, user, org_id}, `${queryName} enter`);
 
-        await validAuth(me, org_id, ACTIONS.READ, TYPES.CLUSTER, queryName, context);
+        let allAllowed = false;
+        try {
+          logger.info({req_id, user, org_id}, `${queryName} validating`);
```

Comment:
'validating' log line should be right after `try {` line so it can 'include' all validation lines.  There arent any other validations here (no specific input values to validate) but should still be consistent.